src/components/Charts/ChartFutureHospitalization.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { ReactNode } from 'react';
+import moment from 'moment';
+import { last, isDate } from 'lodash';
+import { min as d3min, max as d3max } from 'd3-array';
+import { AxisBottom, AxisLeft } from '@vx/axis';
+import { LinePath } from '@vx/shape';
+import { ParentSize } from '@vx/responsive';
+import { scaleLinear, scaleTime } from '@vx/scale';
+import { Projections } from 'common/models/Projections';
+import { COLORS } from 'common';
+import BoxedAnnotation from './BoxedAnnotation';
+import ChartContainer from './ChartContainer';
+import RectClipGroup from './RectClipGroup';
+import Tooltip from './Tooltip';
+import { LegendMarker, LegendLine } from './Legend';
+import * as Style from './Charts.style';
+import { formatDate, formatInteger } from './utils';
+import palette from 'assets/theme/palette';
+
+type Point = {
+  x: number;
+  y: number;
+};
+
+type PointTooltip = Point & {
+  color: string;
+  name: string;
+};
+
+const getDate = (p: Point) => new Date(p.x);
+const getY = (p: Point) => p.y;
+const hasData = (d: any) => isDate(getDate(d)) && Number.isFinite(getY(d));
```

Comment:
Not yet, because the points in the dataset could have `null` or `undefined` values - the idea of this function is to filter out data that is not a `Point`

https://github.com/covid-projections/covid-projections/pull/822/files#diff-def6ce2c3c90cde92f1ea7ff7fc1e832R70-R73

Reply:
Oh, I see!  Makes sense.